hbase-procedure/src/main/java/org/apache/hadoop/hbase/procedure2/ProcedureExecutor.java

```diff
@@ -733,6 +733,9 @@ public long getKeepAliveTime(final TimeUnit timeUnit) {
    * @param chore the chore to add
    */
   public void addChore(ProcedureInMemoryChore<TEnvironment> chore) {
+    if (chore == null) {
```

Comment:
I think it is usually better to do nullable annotations with issues like this to make the contracts clear at the API level.  It also helps static code analyzers to catch issues like these.

For example you could do @Nullable in this case..

Reply:
OK